docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -249,6 +249,16 @@ kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documen
 ```
 {% endcapture %}
 
+{% capture kube-router %}
+
+Kube-router relies on kube-controll-manager to allocate pod CIDR for the nodes. So you must use kubeadm init with --pod-network-cidr flag.
+
+Kube-router provides pod networking, network policy and high performing IPVS/LVS based service proxy as well.
```

Comment:
typo: performing
Kube-router provides pod networking, network policy, and high-performing IP Virtual Server(IPVS)/Linux Virtual Server(LVS) based service proxy.

Reply:
consistency: `kube-router` vs `Kube-router`